Documentation/ApiOverview/PasswordPolicies/Index.rst

```diff
@@ -155,6 +155,10 @@ equal to the old password. The validator must always be configured with
 the exclude action :php:`\TYPO3\CMS\Core\PasswordPolicy\PasswordPolicyAction::NEW_USER_PASSWORD`,
 because it should be excluded, when a new user account is created.
 
+Third-party validators
```

Comment:
```suggestion
..  _..  _password-policies-third-party:

Third-party validators
```

Please always introduce an anchor when you introduce a headline, even though the other headlines in this chapter don't have any yet. (We can fix that in a follow up).